### PR TITLE
Option list optimize

### PR DESCRIPTION
**Pull request by haild173094**

### CODE REVIEW

File: docs/components/Listbox/WithSearch.vue [hunk 1]
```diff
@@ -34,7 +34,7 @@
         @on-active-option-change="handleActiveOptionChange"
       >
         <template v-if="segmentOptions.length > 0">
-          <template v-for="{label, id, value} in segmentList" :key="id">
+          <template v-for="{label, value} in segmentList">
```
Comment (juzser) [inline docs/components/Listbox/WithSearch.vue:37]: Should we set key by `value`?

File: docs/components/Listbox/WithSearch.vue [hunk 2]
```diff
@@ -63,7 +63,7 @@
 </template>
 
 <script setup lang="ts">
-import { ref, computed } from 'vue';
+import { ref, computed, StyleValue } from 'vue';
```
Comment (juzser) [inline docs/components/Listbox/WithSearch.vue:66]: `type StyleValue` or `type CSSProperties`

File: docs/components/Listbox/WithSearch.vue [hunk 3]
```diff
@@ -192,14 +192,14 @@ const handleFilterSegments = (query: string) => {
   filteredSegments.value = nextFilteredSegments;
 };
 
-const handleQueryChange = (e: Event, value: string) => {
+const handleQueryChange = (value: string) => {
```
Comment (juzser) [inline docs/components/Listbox/WithSearch.vue:195]: Or another way is add `_` as prefix of unused params. 
e.g: `_e`

**Review by juzser — CHANGES_REQUESTED** [review]

File: docs/components/Listbox/WithSearch.vue (hunk 3 above)
Comment (haild173094) [reply docs/components/Listbox/WithSearch.vue:195]: e is unused in function

**Review by juzser — APPROVED** [review]

**Pull request merged**